.gitignore

```diff
@@ -6,6 +6,7 @@
 *.log
 dist
 .idea
+mongo.js
```

Comment:
What is this? (Because it's in the gitignore, it's not in GitHub and I don't have the luxury to examine it myself.